mpf/config_spec.yaml

```diff
@@ -1294,15 +1296,21 @@ segment_displays:
     number: single|str|
     platform: single|str|None
     platform_settings: single|dict|None
+virtual_segment_displays:
+    post_update_events: single|bool|false
+light_segment_displays_device:
+    lights: list|dict(str:machine(lights))|
+    type: single|enum(7segment,bcd,14segment,16segment)|
 segment_display_player:
     __valid_in__: machine, mode, show
     __type__: config_player
     priority: single|int_or_token|0
     text: single|str|None
-    color: list|color|ffffff
+    color: list|color|None
     action: single|enum(add,remove,flash,no_flash,flash_match,set_color)|add
     key: single|str|None
     expire: single|ms_or_token|None
+    platform_options: single|dict|None
```

Comment:
Why do we need this in the player? Could we move platform_options (or platform_setting how we call it in most places I think) to the segment_display (simiar to coils/switches)?

Reply:
The thought was to send addition commands to the display at runtime (such as transition effects). Would it be better to add those to the segment_display_player spec directly and then allow other platform implementations to ignore them if not supported (like color)? I'm happy to handle it that way instead. Setting them in the segment_display won't work since it is a runtime option related to the new text.

Reply:
I also could add the transition effects directly to the segment display player since it would then be available to all hardware displays as well...  This is probably a better idea anyway. I will remove the platform_options.